### PR TITLE
feat: add tailwind and custom components

### PR DESCRIPTION
**Pull request by zoheb391**
closes WEB-440

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -13,7 +13,11 @@
     "moduleResolution": "node",
     "resolveJsonModule": true,
     "isolatedModules": true,
-    "jsx": "preserve"
+    "jsx": "preserve",
+    "baseUrl": ".",
+    "paths": {
+      "@components": ["./components/index.tsx"]
```
Comment (acdibble) [inline tsconfig.json:19]: to follow what we do in other repos:
```suggestion
      "@/*": ["./*"]
```

File: package.json [hunk 2]
```diff
@@ -18,6 +18,8 @@
   },
   "homepage": "https://github.com/chainflio-io/chainflip-docs#readme",
   "dependencies": {
+    "@chainflip-io/shared": "^2023.6.7",
```
Comment (acdibble) [inline package.json:21]: You'll have to add some info (like in the bouncer readme) to the readme here on how to get a GH token to install this package. I would just copy the three icons into this repo to prevent this issue.

File: styles/global.css
Comment (acdibble) [inline styles/global.css:1]: this is `globals.css` in our other repos

File: tailwind.config.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  mode: "jit",
```
Comment (acdibble) [inline tailwind.config.js:3]: not sure this is needed anymore, `jit` should be the default

File: package.json (hunk 2 above)
Comment (niklasnatter) [reply package.json:21]: i think we hope for community contributions so i would copy the icons over to reduce friction

File: tsconfig.json [hunk 4]
```diff
@@ -13,7 +13,11 @@
     "moduleResolution": "node",
     "resolveJsonModule": true,
     "isolatedModules": true,
-    "jsx": "preserve"
+    "jsx": "preserve",
+    "baseUrl": ".",
+    "paths": {
+      "@*": ["./*"]
```
Comment (acdibble) [inline tsconfig.json:19]: Needs the slash to prevent if from doing e.g. `@chainflip/sdk` to `./chainflip/sdk`. or if we don't want to use the slash here, we have to revert the change to manually alias all directories
```suggestion
      "@/*": ["./*"]
```

File: pages/integration/swapping-and-aggregation/javascript-sdk/swap-assets/introduction.mdx [hunk 5]
```diff
@@ -12,6 +12,8 @@ There are two main ways to initiate a swap:
 2. **Calling the Vault smart contracts directly**, via the `swap()` or `call()` functions — for EVM-only chains. See [Execute Swap](execute-swap.md).
 
 <Callout type="info">
-  If you haven't yet, make sure you read [How Swapping Workds](../../how-swapping-works.md)
-  and [Swapping Basics](../../swapping-basics.md) to better understand how to execute your first swap.
+  If you haven't yet, make sure you read [How Swapping
+  Workds](../../how-swapping-works.md) and [Swapping
```
Comment (niklasnatter) [inline pages/integration/swapping-and-aggregation/javascript-sdk/swap-assets/introduction.mdx:16]:
```suggestion
  Works](../../how-swapping-works.md) and [Swapping
```

**Review by niklasnatter — APPROVED** [review]

File: pages/integration/swapping-and-aggregation/javascript-sdk/swap-assets/introduction.mdx (hunk 5 above)
Comment (zoheb391) [reply pages/integration/swapping-and-aggregation/javascript-sdk/swap-assets/introduction.mdx:16]: 🦅 👁️
